types/wordpress__components/wordpress__components-tests.tsx

```diff
@@ -288,7 +288,7 @@ const buttonGroupRef = createRef<HTMLDivElement>();
 // dropdown-menu
 //
 <C.DropdownMenu icon="move" label="Select a direction">
-    {({ onClose }) => (
+    {({ onClose }: { onClose: () => void; }) => (
```

Comment:
Ftr, I don't like that I had to do this and this doesn't seem right to me.

[Here's the test failure that inspired this change.](https://github.com/DefinitelyTyped/DefinitelyTyped/actions/runs/4770903812/jobs/8482493345#step:6:1894)

Reply:
You can revert that for now. The [issue is being tracked on the TS repo](https://github.com/microsoft/TypeScript/issues/53941) and I can always merge DT PRs with red CI (packages will be published anyway)